mev-boost-rs/src/service.rs

```diff
@@ -1,10 +1,10 @@
 use crate::relay_mux::RelayMux;
-use ethereum_consensus::{networks, state_transition::Context};
+use ethereum_consensus::{networks, networks::Network, state_transition::Context};
```

Comment:
you'll need to go edit each of these

or maybe try to run `cargo fmt`? I feel like my editor naturally makes this transformation

Reply:
Okay I'll edit every one of these. I guess `cargo fmt` will break the CI since we're using `nightly` to format